device/alphanetworks/x86_64-alphanetworks_snh60a0_320fv2-r0/plugins/led_control.py

```diff
@@ -186,10 +186,14 @@ def _port_name_to_qsfp_index(self, port_name):
 
         lanes = swss.get(
             swss.APPL_DB, self.PORT_TABLE_PREFIX + port_name, 'lanes')
+        lanes_len = 0
```

Comment:
>lanes_len = 0 [](http://example.com/codeflow?start=8&length=13)

minor improvement: move this line into else branch.

Reply:
Fixed